### ISSUE TITLE
Issues with RTL languages

### ISSUE THREAD

**Issue by Orenico10** [easyops-cn/docusaurus-search-local#423]
I tried using your theme on docusaurus 3.4.0, I set the docs direction to be RTL but it seems to break how the searchbar looks, is it possible to manually fix it without breaking your theme?

**Comment by weareoutman** [comment]
Can you share a link or screenshot?

**Comment by Orenico10** [comment]
> Can you share a link or screenshot?

I changed the docusaurus to be RTL and the search bar is doing this weird behavior once I hover over it
![20240619_163050.jpg](https://github.com/easyops-cn/docusaurus-search-local/assets/78265216/15d2385a-7346-4b54-81a3-86490815c1c5)

This is the best screenshot I can get because I'm running it on an offline environment.

**Comment by weareoutman** [comment]
What's the relevant docusaurus config or css styles like, that you implement RTL?

**Comment by weareoutman** [comment]
OK, I reproduced the issue by setting:

```js
i18n: {
  // ...
  localeConfigs: {
      en: {
        direction: "rtl",
      }
    }
}
```

**Comment by weareoutman** [comment]
I'm not familiar with RTL languages, especially how to render a text input, is it ok to make the search bar appear the same as LTR?

<img width="471" alt="image" src="https://github.com/easyops-cn/docusaurus-search-local/assets/2338946/78b965cc-a87e-4922-b9c2-2b7b59d884ee">

**Comment by Orenico10** [comment]
> I'm not familiar with RTL languages, especially how to render a text input, is it ok to make the search bar appear the same as LTR?
> 
> <img width="471" alt="image" src="https://github.com/easyops-cn/docusaurus-search-local/assets/2338946/78b965cc-a87e-4922-b9c2-2b7b59d884ee">
> 

When using RTL languages the search bar is usually mirrored (in comparison to LTR) so the magnifying glass is on the opposite side and the text is forced RTL so any css configurations will set the dir/direction to be rtl rather than ltr/auto, but since this project is for a search bar for docusaurus I suppose keeping it the same as LTR would be fine, as long as I don't get the weird behavior, for now I'm running a script on my website that forces the input's dir to be rtl when you hover over it (basically canceling whatever causes this glitch)

**Comment by weareoutman** [comment]
Fixed in 0.44.2
